### ISSUE TITLE
install behind  proxy failed. 

### ISSUE THREAD

**Issue by pelyxp** [TryGhost/Ghost-CLI#211]
searched doc and google , could not find how to setup proxy for ghost-cli. 
or any offline install method?

before run ghost install , I set following parameters:
```
export http_proxy=proxy-server-name:80
export https_proxy=proxy-server-name:80
export HTTP_PROXY=proxy-server-name:80
export HTTPS_PROXY=proxy-server-name:80
npm config set strict-ssl false -g
npm config set proxy http://proxy-server-name:80 -g
npm config set https-proxy http://proxy-server-name:80 -g
yarn config set proxy http://proxy-server-name:80 -g
yarn config set https-proxy http://proxy-server-name:80 -g
```

```
$ghost install
 ✔ Checking for latest Ghost version
 ✔ Running system checks
 ✔ Setting up install directory
 ❯ Downloading and installing Ghost v1.0.0-beta.1
   ✔ Getting download information
   ✖ Downloading
     → tunneling socket could not be established, cause=connect EINVAL 0.0.0.80:80 - Local (0.0.0.0:0)
     Installing dependencies
   Moving files
An error occurred.
Message: 'tunneling socket could not be established, cause=connect EINVAL 0.0.0.80:80 - Local (0.0.0.0:0)'


Debug Information:
    Node Version: v4.8.3
    Ghost-CLI Version: 1.0.0-alpha.18
    Environment: production
    Command: 'ghost install'
```

```
 more ghost-cli-debug-2017-06-14T07\:33\:11.481Z.log
Debug Information:
    Node Version: v4.8.3
    Ghost-CLI Version: 1.0.0-alpha.18
    Environment: production
    Command: 'ghost install'
An error occurred.
Message: 'tunneling socket could not be established, cause=connect EINVAL 0.0.0.80:80 - Local (0.0.0.0:0)'

Stack: RequestError: tunneling socket could not be established, cause=connect EINVAL 0.0.0.80:80 - Local (0.0.0.0:0)
    at ClientRequest.<anonymous> (/root/.nvm/v4.8.3/lib/node_modules/ghost-cli/node_modules/download/node_modules/got/index.js:73:21)
    at ClientRequest.g (events.js:260:16)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at ClientRequest.onError (/root/.nvm/v4.8.3/lib/node_modules/ghost-cli/node_modules/download/node_modules/caw/node_modules/tunnel-agent/index.js:178:21)
    at ClientRequest.g (events.js:260:16)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at Socket.socketErrorListener (_http_client.js:269:9)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at emitErrorNT (net.js:1272:8)
    at nextTickCallbackWith2Args (node.js:511:9)
    at process._tickCallback (node.js:425:17)
Code: ECONNRESET
Path: /ghost/-/ghost-1.0.0-beta.1.tgz
```

**Comment by acburdine** [comment]
Hmmm.... this is a use case I hadn't considered before. Ghost-CLI gets its download link from `yarn info`, but I guess that wouldn't contain any information as to whether or not a proxy was needed to connect.

As there's not much other need for Ghost-CLI to have a global config, my thought is to use environment variables and/or command line flags to tell the CLI that it needs to connect via a proxy. (Now all I have to do is figure out how to use the proxy data 😛 )

**Comment by acburdine** [comment]
@pelyxp So as it turns out - this isn't a bug in Ghost-CLI itself but rather a bug in the underlying download module (https://github.com/kevva/download/issues/64). Based on my reading of that issue, it seems that it can be resolved by either:
a) Adding a protocol to the proxy server url
b) Setting the env var `HTTPS_PROXY=''` and leaving HTTP_PROXY

Can you try one or both of those and see if you can get it working?

**Comment by pelyxp** [comment]
@acburdine  thanks for update, I tried following , now ghost install worked. 

```
export http_proxy=http://proxy-server:80
export https_proxy=http://proxy-server:80
```
worked
<br>
```
export HTTP_PROXY=proxy-server:80
export HTTPS_PROXY=''
```
not work
<br>
```
export HTTP_PROXY=http://proxy-server:80
export HTTPS_PROXY=''
```
worked

**Comment by acburdine** [comment]
@kirrg001 Based on the fact that this is an issue in another project this probably just needs to be documented under some sort of "troubleshooting" section.

**Comment by kirrg001** [comment]
@acburdine agree 👍

**Comment by kirrg001** [comment]
I have referenced this issue in the documentation section in #216.

**Comment by acburdine** [comment]
This has been noted in the docs at https://docs.ghost.org , closing.

**Comment by lucistanescu** [comment]
It doesn't look like setting setting the `http_proxy` and `https_proxy` environment variables or setting the npm `proxy` and `https-proxy` config options do the trick as this thread or the documention point out.

I'm trying to install ghost on a system that can only access a proxy via IPv6. I installed a local squid proxy because npm couldn't take a link-local IPv6 address for the `proxy` options, which means either `http://[fe80::bla%eth0]:3128` or `http://127.0.0.1:3128` work on this machine.

However, running a tcpdump clearly shows that ghost-cli (well, whatever underlying library it's using) is adamant to do a DNS query and try to connect to registry.npmjs.org (port 80 or 443 depending on the `registry` npm setting):

    18:50:00.549680 IP 127.0.0.1.45093 > 127.0.0.1.53: 62755+ AAAA? registry.npmjs.org. (36)
    18:50:00.550260 IP 127.0.0.1.53 > 127.0.0.1.45093: 62755- 1/0/0 AAAA ::1 (82)
    18:50:00.551045 IP6 ::1.54332 > ::1.80: Flags [S], seq 687298623, win 43690, options [mss 65476,sackOK,TS val 2121869 ecr 0,nop,wscale 7], length 0
    18:50:00.551063 IP6 ::1.80 > ::1.54332: Flags [R.], seq 0, ack 687298624, win 0, length 0

(After I noticed the DNS queries, the system is replying with a fake address of ::1 to AAAA queries).

Output of `ghost install` is:

    $ ghost install --db sqlite3
    An error occurred.
    Message: 'connect ECONNREFUSED ::1:80'
    
    
    Debug Information:
        Node Version: v6.12.3
        Ghost-CLI Version: 1.4.2
        Environment: production
        Command: 'ghost install --db sqlite3'
    
    Additional log info available in: /home/ubuntu/.ghost/logs/ghost-cli-debug-2018-02-05T18_51_00_741Z.log
    
    Please refer to https://docs.ghost.org/v1/docs/troubleshooting#section-cli-errors for troubleshooting.

Debug file content is:

    Debug Information:
        Node Version: v6.12.3
        Ghost-CLI Version: 1.4.2
        Environment: production
        Command: 'ghost install --db sqlite3'
    An error occurred.
    Message: 'connect ECONNREFUSED ::1:80'
    
    Stack: RequestError: connect ECONNREFUSED ::1:80
        at ClientRequest.req.once.err (/usr/lib/node_modules/ghost-cli/node_modules/package-json/node_modules/got/index.js:73:21)
        at ClientRequest.g (events.js:292:16)
        at emitOne (events.js:96:13)
        at ClientRequest.emit (events.js:188:7)
        at Socket.socketErrorListener (_http_client.js:310:9)
        at emitOne (events.js:96:13)
        at Socket.emit (events.js:188:7)
        at emitErrorNT (net.js:1281:8)
        at _combinedTickCallback (internal/process/next_tick.js:80:11)
        at process._tickCallback (internal/process/next_tick.js:104:9)
    Code: ECONNREFUSED
    Path: /ghost-cli

**Comment by aemonge** [comment]
# This issue is **still on going with the latest cli # ## version:**

```
 ⚙ aemonge   ~/t/ghost   HTTPS_PROXY=''; ghost i local -v
An error occurred.
Message: 'getaddrinfo ENOTFOUND registry.npmjs.org registry.npmjs.org:443'


Debug Information:
    OS: Arch, vrolling
    Node Version: v8.9.4
    Ghost-CLI Version: 1.6.0
    Environment: production
    Command: 'ghost i local -v'
```

**prove npm is working through the proxy**
```
 ✘ aemonge   ~/t/ghost   npm i ghost-cli
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but npm-shrinkwrap.json was generated for lockfileVersion@0. I'll try to do my best with it!
npm WARN ghost@1.0.0 No description
npm WARN ghost@1.0.0 No repository field.

+ ghost-cli@1.7.1
added 71 packages, removed 47 packages and updated 356 packages in 18.167s

```

**Prove that Yarn is working through the proxy**
```
 aemonge   ~/t/ghost   yarn install --save ghost
yarn install v1.5.1
error An unexpected error occurred: "`install` has been replaced with `add` to add new dependencies. Run \"yarn add ghost\" instead.".
info If you think this is a bug, please open a bug report with the information provided in "/home/aemonge/tmp/ghost/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 ✘ aemonge   ~/t/ghost   yarn add --save ghost
yarn add v1.5.1
error An unexpected error occurred: "Missing list of packages to add to your project.".
info If you think this is a bug, please open a bug report with the information provided in "/home/aemonge/tmp/ghost/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
 ✘ aemonge   ~/t/ghost   yarn init
yarn init v1.5.1
question name (ghost):
question version (1.0.0):
question description:
question entry point (index.js):
question repository url:
question author:
question license (ISC):
question private:
success Saved package.json
Done in 2.05s.
 aemonge   ~/t/ghost  
 aemonge   ~/t/ghost  
 aemonge   ~/t/ghost   yarn add --save ghost
yarn add v1.5.1
error An unexpected error occurred: "Missing list of packages to add to your project.".
info If you think this is a bug, please open a bug report with the information provided in "/home/aemonge/tmp/ghost/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
 ✘ aemonge   ~/t/ghost   yarn install --save ghost
yarn install v1.5.1
error An unexpected error occurred: "`install` has been replaced with `add` to add new dependencies. Run \"yarn add ghost\" instead.".
info If you think this is a bug, please open a bug report with the information provided in "/home/aemonge/tmp/ghost/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 ✘ aemonge   ~/t/ghost   yarn add ghost
yarn add v1.5.1
[1/4] Resolving packages...
warning ghost > nodemailer@0.7.1: All versions below 4.0.1 of Nodemailer are deprecated. See https://nodemailer.com/status/
warning ghost > nodemailer > mailcomposer@0.2.12: This project is unmaintained
warning ghost > jsonpath > jison > nomnom@1.5.2: Package no longer supported. Contact support@npmjs.com for more info.
warning ghost > jsonpath > jison > jison-lex > nomnom@1.5.2: Package no longer supported. Contact support@npmjs.com for more info.
warning ghost > nodemailer > mailcomposer > mimelib@0.2.19: This project is unmaintained
warning ghost > nodemailer > mailcomposer > dkim-signer > mimelib@0.2.19: This project is unmaintained
[2/4] Fetching packages...
warning ghost@1.22.2: The engine "cli" appears to be invalid.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 339 new dependencies.
info Direct dependencies
└─ ghost@1.22.2
info All dependencies
├─ @segment/loosely-validate-event@1.1.2
├─ accepts@1.3.5
├─ addressparser@0.3.2
├─ ajv@5.5.2
├─ align-text@0.1.4
├─ amdefine@1.0.1
├─ amperize@0.3.7
├─ analytics-node@2.4.1
├─ append-field@0.1.0
├─ aproba@1.2.0
├─ archiver@1.3.0
├─ are-we-there-yet@1.1.4
├─ argparse@1.0.10
├─ arr-flatten@1.1.0
├─ array-each@1.0.1
├─ array-flatten@1.1.1
├─ array-slice@1.1.0
├─ array-uniq@1.0.3
├─ asn1@0.2.3
├─ assign-symbols@1.0.0
├─ asynckit@0.4.0
├─ atob@2.1.0
├─ aws-sdk-apis@3.1.10
├─ aws-sdk@2.0.5
├─ aws-sign2@0.7.0
├─ aws4@1.7.0
├─ base@0.11.2
├─ bcrypt-pbkdf@1.0.1
├─ bcryptjs@2.4.3
├─ bookshelf-relations@0.2.0
├─ bookshelf@0.13.3
├─ boom@4.3.1
├─ braces@2.3.2
├─ brute-knex@2.0.0
├─ bson-objectid@1.2.2
├─ bunyan-loggly@1.3.3
├─ bunyan@1.8.5
├─ busboy@0.2.14
├─ cache-base@1.0.1
├─ caller@1.0.1
├─ caseless@0.12.0
├─ center-align@0.1.3
├─ cheerio@0.22.0
├─ chownr@1.0.1
├─ cjson@0.2.1
├─ class-utils@0.3.6
├─ clone@2.1.2
├─ co@4.6.0
├─ collection-visit@1.0.0
├─ combined-stream@1.0.6
├─ component-type@1.2.1
├─ compress-commons@1.2.2
├─ compressible@2.0.13
├─ compression@1.7.2
├─ concat-stream@1.6.0
├─ connect-slashes@1.3.1
├─ console-control-strings@1.1.0
├─ cookie-session@1.3.2
├─ cookie-signature@1.0.6
├─ cookie@0.3.1
├─ cookiejar@2.1.1
├─ cookies@0.7.1
├─ copy-descriptor@0.1.1
├─ core-js@2.5.5
├─ cors@2.8.4
├─ crc@3.5.0
├─ crc32-stream@2.0.0
├─ create-error@0.3.1
├─ cryptiles@3.1.2
├─ crypto-token@1.0.1
├─ css-select@1.2.0
├─ css-what@2.1.0
├─ csv-parser@1.12.0
├─ dashdash@1.14.1
├─ deep-is@0.1.3
├─ delayed-stream@1.0.0
├─ delegates@1.0.0
├─ destroy@1.0.4
├─ detect-file@1.0.0
├─ detect-libc@1.0.3
├─ dicer@0.2.5
├─ directmail@0.1.8
├─ dkim-signer@0.1.2
├─ domhandler@2.4.1
├─ domutils@1.5.1
├─ downsize@0.0.8
├─ dtrace-provider@0.8.6
├─ ebnf-parser@0.1.10
├─ ecc-jsbn@0.1.1
├─ editorconfig@0.13.3
├─ ee-first@1.1.1
├─ emits@3.0.0
├─ encoding@0.1.12
├─ entities@1.1.1
├─ escodegen@0.0.21
├─ esprima@1.0.4
├─ estraverse@4.2.0
├─ esutils@2.0.2
├─ expand-brackets@2.1.4
├─ express-brute@1.0.1
├─ express-hbs@1.0.4
├─ express@4.16.3
├─ extend@3.0.1
├─ extglob@2.0.4
├─ extract-zip@1.6.6
├─ extsprintf@1.3.0
├─ fast-deep-equal@1.1.0
├─ fast-json-stable-stringify@2.0.0
├─ fast-levenshtein@2.0.6
├─ fill-range@4.0.0
├─ finalhandler@1.1.1
├─ find-root@1.0.0
├─ findup-sync@2.0.0
├─ fined@1.1.0
├─ flagged-respawn@1.0.0
├─ follow-redirects@0.0.3
├─ for-in@1.0.2
├─ forever-agent@0.6.1
├─ form-data@2.3.2
├─ formidable@1.2.1
├─ forwarded@0.1.2
├─ fs-minipass@1.2.5
├─ gauge@2.7.4
├─ getpass@0.1.7
├─ ghost-gql@0.0.9
├─ ghost-storage-base@0.0.1
├─ ghost@1.22.2
├─ gscan@1.4.0
├─ handlebars@4.0.6
├─ har-schema@2.0.0
├─ har-validator@5.0.3
├─ has-unicode@2.0.1
├─ has-value@1.0.0
├─ has-values@1.0.0
├─ hawk@6.0.2
├─ hijackresponse@2.0.1
├─ html-to-text@3.3.0
├─ htmlparser2@3.9.2
├─ http-signature@1.2.0
├─ ignore-walk@3.0.1
├─ image-size@0.6.2
├─ inflection@1.12.0
├─ intl-messageformat-parser@1.2.0
├─ intl-messageformat@1.3.0
├─ intl@1.2.5
├─ ipaddr.js@1.6.0
├─ is-absolute@1.0.0
├─ is-accessor-descriptor@1.0.0
├─ is-buffer@1.1.6
├─ is-data-descriptor@1.0.0
├─ is-descriptor@1.0.2
├─ is-extglob@2.1.1
├─ is-glob@3.1.0
├─ is-odd@2.0.0
├─ is-plain-object@2.0.4
├─ is-relative@1.0.0
├─ is-typedarray@1.0.0
├─ is-unc-path@1.0.0
├─ isstream@0.1.2
├─ jison-lex@0.2.1
├─ jison@0.4.13
├─ join-component@1.1.0
├─ js-beautify@1.6.8
├─ json-schema-traverse@0.3.1
├─ json-schema@0.2.3
├─ jsonpath@1.0.0
├─ JSONSelect@0.4.0
├─ jsprim@1.4.1
├─ keygrip@1.0.2
├─ kind-of@3.2.2
├─ knex-migrator@3.1.5
├─ knex@0.14.4
├─ lazy-cache@1.0.4
├─ lazystream@1.0.0
├─ levn@0.3.0
├─ lex-parser@0.1.4
├─ linkify-it@2.0.3
├─ lodash.assignin@4.2.0
├─ lodash.bind@4.2.1
├─ lodash.clonedeep@4.5.0
├─ lodash.defaults@4.2.0
├─ lodash.escaperegexp@4.1.2
├─ lodash.filter@4.6.0
├─ lodash.flatten@4.4.0
├─ lodash.foreach@4.5.0
├─ lodash.isplainobject@4.0.6
├─ lodash.isstring@4.0.1
├─ lodash.map@4.6.0
├─ lodash.merge@4.6.1
├─ lodash.mergewith@4.6.1
├─ lodash.pick@4.4.0
├─ lodash.reduce@4.6.0
├─ lodash.reject@4.6.0
├─ lodash.some@4.6.0
├─ lodash.unescape@4.0.1
├─ long-timeout@0.1.1
├─ longest@1.0.1
├─ mailcomposer@0.2.12
├─ make-iterator@1.0.1
├─ map-visit@1.0.0
├─ markdown-it-footnote@3.0.1
├─ markdown-it-lazy-headers@0.1.3
├─ markdown-it-mark@2.0.0
├─ markdown-it@8.4.1
├─ mdurl@1.0.1
├─ media-typer@0.3.0
├─ merge-descriptors@1.0.1
├─ methods@1.1.2
├─ micromatch@3.1.10
├─ mime-types@2.1.18
├─ mime@1.6.0
├─ minizlib@1.1.0
├─ mixin-deep@1.3.1
├─ mobiledoc-dom-renderer@0.6.5
├─ moment-timezone@0.5.14
├─ moment@2.22.0
├─ multer@1.3.0
├─ mv@2.1.1
├─ nan@2.9.2
├─ nanomatch@1.2.9
├─ ncp@2.0.0
├─ needle@2.2.0
├─ negotiator@0.6.1
├─ netjet@1.3.0
├─ node-loggly-bulk@2.2.2
├─ node-pre-gyp@0.9.0
├─ nodemailer@0.7.1
├─ nopt@4.0.1
├─ npm-bundled@1.0.3
├─ npm-packlist@1.1.10
├─ npmlog@4.1.2
├─ nth-check@1.0.1
├─ oauth-sign@0.8.2
├─ oauth2orize@1.11.0
├─ object-copy@0.1.0
├─ object.defaults@1.1.0
├─ object.map@1.0.1
├─ optionator@0.8.2
├─ os-homedir@1.0.2
├─ osenv@0.1.5
├─ parse-filepath@1.0.2
├─ pascalcase@0.1.1
├─ passport-http-bearer@1.0.1
├─ passport-oauth2-client-password@0.1.2
├─ passport@0.4.0
├─ path-match@1.2.4
├─ path-parse@1.0.5
├─ path-root-regex@0.1.2
├─ path-root@0.1.1
├─ path-to-regexp@0.1.7
├─ pause@0.0.1
├─ performance-now@2.1.0
├─ posix-character-classes@0.1.1
├─ postcss@6.0.21
├─ posthtml-parser@0.2.1
├─ posthtml-render@1.1.3
├─ posthtml@0.9.2
├─ proxy-addr@2.0.3
├─ public-address@0.1.2
├─ punycode@1.2.4
├─ qs@6.5.1
├─ rai@0.1.12
├─ raw-body@2.3.2
├─ readdirp@2.1.0
├─ rechoir@0.6.2
├─ regenerator-runtime@0.11.1
├─ remove-trailing-separator@1.1.0
├─ remove-trailing-slash@0.1.0
├─ repeat-element@1.1.2
├─ request@2.85.0
├─ require-dir@0.3.2
├─ resolve-url@0.2.1
├─ resolve@1.7.1
├─ ret@0.1.15
├─ right-align@0.1.3
├─ rss@1.2.2
├─ safe-json-stringify@1.1.0
├─ sanitize-html@1.18.2
├─ sax@0.4.2
├─ secure-keys@1.0.0
├─ serve-static@1.13.2
├─ set-immediate-shim@1.0.1
├─ set-value@2.0.0
├─ sigmund@1.0.1
├─ simple-dom@0.3.2
├─ simple-html-tokenizer@0.4.3
├─ simplesmtp@0.3.35
├─ snapdragon-node@2.1.1
├─ snapdragon-util@3.0.1
├─ sntp@2.1.0
├─ source-map-resolve@0.5.1
├─ source-map-url@0.4.0
├─ source-map@0.5.7
├─ split-string@3.1.0
├─ sprintf-js@1.1.1
├─ sqlite3@4.0.0
├─ srcset@1.0.0
├─ sshpk@1.14.1
├─ static-eval@2.0.0
├─ static-extend@0.1.2
├─ streamsearch@0.1.2
├─ stringstream@0.0.5
├─ superagent-retry@0.6.0
├─ superagent@3.8.2
├─ tar@4.4.1
├─ tarn@1.1.4
├─ to-regex-range@2.1.1
├─ tough-cookie@2.3.4
├─ tweetnacl@0.14.5
├─ type-is@1.6.16
├─ uc.micro@1.0.5
├─ uglify-js@2.8.29
├─ uglify-to-browserify@1.0.2
├─ uid2@0.0.3
├─ unc-path-regex@0.1.2
├─ underscore.string@3.3.4
├─ underscore@1.8.3
├─ unidecode@0.1.8
├─ union-value@1.0.0
├─ unpipe@1.0.0
├─ unset-value@1.0.0
├─ urix@0.1.0
├─ use@3.1.0
├─ utils-merge@1.0.1
├─ uuid@3.2.1
├─ v8flags@3.0.2
├─ validator@6.3.0
├─ verror@1.10.0
├─ walkdir@0.0.11
├─ wide-align@1.1.2
├─ window-size@0.1.4
├─ wordwrap@0.0.3
├─ xml2js@0.2.6
├─ xmlbuilder@0.4.2
├─ xoauth2@0.1.8
├─ xregexp@2.0.0
├─ yallist@3.0.2
├─ yauzl@2.4.1
└─ zip-stream@1.2.0
Done in 197.70s.

```


I've tried the above mentioned of disabling the httpS proxy, but it didn't helped. Any other suggestions?

**Comment by kirrg001** [comment]
@aemonge I would suggest you post your problem in our [forum](https://forum.ghost.org). Maybe somebody has an idea. I doubt this is caused by the CLI, but we can leave this issue open till we know.

**Comment by cheahkhing** [comment]
Hi, this is definitely an issue currently impacting ghost.
I was getting hit by this as well.
After some investigations, it seems to be rooted from "package-json" module, where it uses "got" module and didn't take care of any proxy settings. :/

if you really want to make your install works for the time being, you can do so:
goto your ghost-cli module, find the "package-json" folder inside its node_modules folder:
`npm install tunnel`
edit "index.js":
add:
`
const tunnel = require('tunnel');

line 57
	const gotOptions = {
		json: true,
		headers,
		agent: tunnel.httpsOverHttp({
			proxy: {
				host: 'yourproxy.com',
				port: 912
			}
		})
	};
`
Now it should work properly.
I shall try to submit a PR for package-json when i get some time later.

**Comment by zicklag** [comment]
Hey @cheahkhing that worked for me too! Thanks for the workaround. I think it would be great if there could be a fix implemented for this.

**Comment by cheahkhing** [comment]
Hi, @zicklag I've created a pull request on this, however it seems like this project is quite 'inactive' for sometime since a lot of pull requests not being merged. :/
Hopefully it won't die out soon.

**Comment by acburdine** [comment]
Hey everyone! Apologies for the delay on getting this implemented, support for installing Ghost behind proxies is implemented and will be released as part of Ghost-CLI 1.13.0 😄 

Note: support being added is basic support for the proxy environment variables (i.e. http_proxy/https_proxy). Additional options such as custom CA certificates for proxying are a bit more complicated and aren't supported yet.

**Comment by acburdine** [comment]
Released as Ghost-CLI 1.13.0.
